tests/PhpSpreadsheetTests/Calculation/Functions/Web/HttpClientMock.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace PhpOffice\PhpSpreadsheetTests\Calculation\Functions\Web;
+
+use Closure;
+use Psr\Http\Client\ClientInterface;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+
+class HttpClientMock implements ClientInterface
```

Comment:
This entire class should be replaced by PHPunit mock mechanic in order to also test that the method is called exactly once.

Reply:
I don't see any value in checking the number of times the client is tested.

I could change the mock to remove the internal response when `sendRequest()` is used, this will ensure it's used only once.

Also, using phpunit mocks has a performance impact. Here, having a custom mock is costless